src/main.cpp

```diff
@@ -258,11 +268,16 @@ int main(int argc, char **argv) {
     } else {
       fprintf(stderr, "Error opening handler for device %s\n",
               current_dev->name);
+      ++nb_failed_devices;
     }
 
     current_dev = current_dev->next;
   }
 
+  if (nb_devices == nb_failed_devices) {
+    forceExit(false, "Error opening pcap handlers for all devices.\n");
+  }
```

Comment:
nice

Reply:
Thanks :) I didn't really see that missing until I ran `nethogs -f "foo bar baz"` and saw that due to the failed filter compilation, every handler failed to open... but the UI just sat there doing nothing...